bugbug/feature_cleanup.py

```diff
@@ -198,3 +198,66 @@ def __init__(self):
 
     def __call__(self, text):
         return self.pattern.sub("__CRASH_STATS_LINK__", text)
+
+
+class extract_description(object):
```

Comment:
We recently started using Pascal case for such class names.[^1] Also, since this is used as a cleaning function. It could be more descriptive to avoid using "extract" in the class name.
```suggestion
class CleanCompatibilityReportDescription(object):
```

[^1]: If you update your branch from master, other classes will be using Pascal case as well.